src/courtroom.cpp

```diff
@@ -3321,7 +3321,7 @@ void Courtroom::start_chat_ticking()
   // handle expanded desk mods
   switch(m_chatmessage[DESK_MOD].toInt()) {
     case 4:
-      set_self_offset(m_chatmessage[SELF_OFFSET]);
+      set_self_offset(m_chatmessage[SELF_OFFSET], QString("||"));
```

Comment:
`QString("||")` is not necessary. You may either use `"||"` or `QStringLiteral("||")` (some also prefer `QLatin1String` for efficiency, but the performance benefit is negligible)